### PR TITLE
EPMRPP-76606 || Disable tooltip and message from plugin details

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

File: app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const messages = defineMessages({
   disablePluginMessage: {
     id: 'PluginItem.disablePluginMessage',
     defaultMessage:
-      'Are you sure you want to disable a plugin {pluginName}? If you disable plugin, information about it will be hidden on the Project Settings and users can not interact with it',
+      'Are you sure you want to disable a plugin {pluginName}? If you disable plugin, information about it will be hidden on the {pluginLocation} and users can not interact with it',
```
Comment (AmsterGet) [inline app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx:70]: Please run `manage:translations` to update corresponding messages.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx (hunk 1 above)
Comment (tr1ble) [reply app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx:70]: I ran it, but nothing was changed

Comment (AmsterGet) [reply app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx:70]: Seems like you need to update it manually to support new parameter.

File: app/localization/translated/ru.json [hunk 2]
```diff
@@ -1596,6 +1596,7 @@
   "ServerSettingsTabs.updateAuthSuccess": "Настройки интеграции OAuth обновлены",
   "ServiceVersionItem.newVersion": "Доступна новая версия: {newVersion}",
   "ServiceVersionsBlock.currentVersion": "Текущая версия",
+  "SettingsPage.analysis": "Auto-Analysis",
```
Comment (AmsterGet) [inline app/localization/translated/ru.json:1599]: Should it be translated?

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**